### ISSUE TITLE
CRw: Alternate Text missing in the Pagination

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#272]
![image](https://cloud.githubusercontent.com/assets/7510842/5203616/8f85ebd4-7588-11e4-9520-3c357ad0b4c7.png)

A screen reader in Browse mode (Arrow Buttons) does not read the Forth and back-buttons correctly.

==> Use ARIA "labelledby" or a hidden text.
If you use ARIA check it out with the screen reader NVDA http://www.nvaccess.org/download/

See #249
